src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,4 @@
+package core.basesyntax;
+
+public class Excavator {
```

Comment:
There is a redundant empty line after the class declaration. According to the checklist, please remove unnecessary empty lines.